Cargo.toml

```diff
@@ -93,6 +93,13 @@ flac = ["bevy_internal/flac"]
 mp3 = ["bevy_internal/mp3"]
 vorbis = ["bevy_internal/vorbis"]
 wav = ["bevy_internal/wav"]
+symphonia-aac = ["bevy_internal/symphonia-aac"]
+symphonia-all = ["bevy_internal/symphonia-all"]
+symphonia-flac = ["bevy_internal/symphonia-flac"]
+symphonia-isomp4 = ["bevy_internal/symphonia-isomp4"]
+symphonia-mp3 = ["bevy_internal/symphonia-mp3"]
```

Comment:
Why expose this when there's another mp3 feature already? Same with wav and vorbis.

Reply:
Symphonia is an alternative backend to what rodio uses, which depends on the features used (flac = claxon, vorbis = lewton, wav = hound, mp3 = minimp3).

imo these features should be available because:

1. For completeness sake.
2. When you want to support formats, where is only supported by symphonia (for example `aac` and `wav`) you'll have to compile both hound and symphonia. I think it's better as a user to just enable symphonia's wav feature instead of using hound to reduce compile times.